### PR TITLE
Resolve second task

### PR DESCRIPTION
**Pull request by AleksandrM178**

### CODE REVIEW

**Comment by Aleksimmortal** [comment]
DEVELOPERI!

**Comment by Jeffry1990** [comment]
х=y

File: TestGit19/Program.cs [hunk 1]
```diff
@@ -4,4 +4,9 @@
 
 int k = 12;
 var y = 14 * 12 * k;
-Console.WriteLine(y);
\ No newline at end of file
+Console.WriteLine(y);
+
+int e = 12;
```
Comment (Aleksimmortal) [inline TestGit19/Program.cs:9]: Again C#(((!

File: TestGit19/Program.cs [hunk 2]
```diff
@@ -4,4 +4,9 @@
 
 int k = 12;
 var y = 14 * 12 * k;
-Console.WriteLine(y);
\ No newline at end of file
+Console.WriteLine(y);
```
Comment (inSagen) [inline TestGit19/Program.cs:7]: writeLine

**Review by AleksandrM178 — COMMENTED** [review]
Good

**Pull request merged**